packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-pwd-email/index.md

```diff
@@ -24,33 +24,26 @@ This use case requires the **password** and **email** factors.
 
 ## Configuration updates
 
-Before you build the self-registration flow in your app, configure the Okta org to accept self-registration with both the password and email factors by performing the following steps.
+Before you build the sign-in flow with password and email factors in your app, configure the Okta org to accept both the password and email factors by performing the following steps.
 
-### Step 1: Set up your Okta org for password factor only use cases
+### Step 1: Set up your Okta org for password factor
```

Comment:
> Step 1: Set up your Okta org for a password factor
or
> Step 1: Set up your Okta org to use password factors

Reply:
I used: 
Step 1: Set up your Okta org for a password factor